cirq/google/gate_sets.py

```diff
@@ -83,5 +83,9 @@
         MEASUREMENT_DESERIALIZER,
     ],
 )
-
 document(XMON, """Gate set for XMON devices.""")
+
+NAMED_GATESETS = {
+    'sqrt-iswap': SQRT_ISWAP_GATESET,
```

Comment:
Strange that these names don't match the gate set names (which are used when they are serialized).

Reply:
changed